desktop_files/package.json.orig

```diff
@@ -68,11 +69,11 @@
     "react-native-fs": "git+https://github.com/status-im/react-native-fs.git",
     "react-native-http": "github:tradle/react-native-http#834492d",
     "react-native-http-bridge": "0.6.1",
-    "react-native-i18n": "git+https://github.com/status-im/react-native-i18n.git#version_0.0.8_desktop",
     "react-native-image-crop-picker": "0.18.1",
     "react-native-image-resizer": "1.0.0",
     "react-native-invertible-scroll-view": "1.1.0",
     "react-native-keychain": "git+https://github.com/status-im/react-native-keychain.git",
+    "react-native-languages": "git+https://github.com/kn/react-native-languages.git#version_0.1.1_desktop",
```

Comment:
Blocker: this should be a release/tag, not a branch, so that the build is reproducible in the future.

Reply:
Sorry @kn, I need you to create a release on `react-native-languages` (or we can fork it and create it ourselves) so that we keep the guarantee of reproducible builds (we just added a bot that would have caught this issue in the PR, but slightly too late ;-) )

Reply:
Good point! I think it will be better if you can fork the repo under status-im org as it was for the react-native-i18n for future maintenance. Would you mind doing that for me? Thanks!

Reply:
Thanks! Updated.

Reply:
Awesome, thanks again @kn. Could you please rebase the branch to get rid of the merge conflicts?